projects/ssr-tests/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
+  "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "baseUrl": "./",
```

Comment:
- inheriting baseUrl being root dir - so all `Config` interface augmentations are taken into account (for type safety of `TestConfig`
- inheriting `paths` mapping for all libs (.e.g. `@sparatcus/core` resolved to `projects/core/src/public_api`) - so we can import `TestConfig` from `@spartacus/core`